### PR TITLE
Docs refresh for Contour 0.6-beta.1. Fixes #171

### LINKED ISSUE #171: Update docs to reference IngressRoute
Our [route](https://github.com/heptio/gimbal/blob/master/docs/route.md) documentation should be updated to reference IngressRoutes.  Mostly we can point to the Contour documentation, but we should highlight:

* --root-namespaces
* Delegation

### PR DESCRIPTION
**Pull request by rosskukulinski**
Signed-off-by: Ross Kukulinski <ross@kukulinski.com>

### CODE REVIEW

File: docs/route.md [hunk 1]
```diff
@@ -1,30 +1,50 @@
 # Route Specification
 
-The core of Gimbal is Routes, which allow traffic into one or more applications. This section will discuss how to utilize Kubernetes `Ingress` objects to create these routes. 
+The core of Gimbal is Ingressroutes, which allow traffic to be routed into one or more applications. This section will discuss how to utilize [Contour IngressRoute](https://github.com/heptio/contour/blob/master/docs/ingressroute.md) objects to create these definitions.
```
Comment (alexbrand) [inline docs/route.md]: nit: s/Ingressroutes/IngressRoutes

File: docs/route.md [hunk 2]
```diff
@@ -1,30 +1,50 @@
...
 
 Before beginning it is important to understand how service discovery functions within Gimbal. The Discoverer components should be deployed per upstream cluster. Once synchronized, services will show up in your team namespace with the cluster name appended.
 
-For example, if a Kubernetes cluster is being discovered and there was a service named `s1` which existed in the namespace `team1`, in the cluster `node02`, once synchronized the service in the Gimbal cluster would be named `s1-node02`. 
+For example, if a Kubernetes cluster is being discovered and there was a service named `s1` which existed in the namespace `team1`, in the cluster `node02`, once synchronized the service in the Gimbal cluster would be named `s1-node02`.
```
Comment (stevesloka) [inline docs/route.md]: Services are: `${backend-name}-${service-name}` so I think this example would be: `node02-s1`

File: docs/teams.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Teams with Gimbal
 
-A key feature of Gimbal is team management. Teams should be able to configure and define their own Ingress resources within the Gimbal cluster without requiring an administrator to assist. To enable this, users should be allowed access only to specified namespaces in the Gimbal cluster. Within their respective namespaces, team members should be granted specific authorization to create Ingress routes and to view Services and Endpoints. 
+A key feature of Gimbal is team management. Teams should be able to configure and define their own IngressRoute resources within the Gimbal cluster without requiring an administrator to assist. To enable this, users should be allowed access only to specified namespaces in the Gimbal cluster. Within their respective namespaces, team members should be granted specific authorization to create IngressRoutes and to view Services and Endpoints.
```
Comment (stevesloka) [inline docs/teams.md]: Might be good to mention here that teams need Admins to delegate Virtualhosts to them. It was discussed earlier in delegate section.

Comment (rosskukulinski) [reply docs/teams.md]: done

File: docs/route.md [hunk 4]
```diff
@@ -1,30 +1,50 @@
...
+The core of Gimbal is IngressRoutes, which allow traffic to be routed into one or more applications. This section will discuss how to utilize [Contour IngressRoute](https://github.com/heptio/contour/blob/master/docs/ingressroute.md) objects to create these definitions.
 
 Before beginning it is important to understand how service discovery functions within Gimbal. The Discoverer components should be deployed per upstream cluster. Once synchronized, services will show up in your team namespace with the cluster name appended.
 
-For example, if a Kubernetes cluster is being discovered and there was a service named `s1` which existed in the namespace `team1`, in the cluster `node02`, once synchronized the service in the Gimbal cluster would be named `s1-node02`. 
+For example, if a Kubernetes cluster is being discovered and there was a service named `s1` which existed in the namespace `team1`, in the cluster `cluster1`, once synchronized, the service in the Gimbal cluster would be named `cluster1-s1` and it would be in the `team` namespace.
 
-## Basic Route
+## Basic Routex
```
Comment (stevesloka) [inline docs/route.md]: s/Routex/Route

**Pull request merged**